docs/agent-format.md

```diff
@@ -286,7 +286,8 @@ For detailed information about hook behavior, input/output formats, and examples
     "postToolUse": [
       {
         "matcher": "fs_write",
-        "command": "cargo fmt --all"
+        "command": "cargo fmt --all",
+        "only_when_turn_complete": true
```

Comment:
`onlyWhenTurnComplete` -> Can you please  Camel case consistent with rest of the config ?